.github/workflows/update_news_zenodo.yml

```diff
@@ -0,0 +1,21 @@
+name: Update repo NEWS and zenodo files on approval of pull request
+
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
```suggestion
    types: [submitted]
    branches:
     - main
```

Reply:
I found it in an older version of testprotocol, so we used it that way before we decided on putting both dockers together in the same GHA.  But I don't know if it worked, I guess I haven't tested it (other than the fact that it caused no problems in a PR to main).
Anyway, I leave it up to you to decide if it is necessary to add it: will there ever be a branch named after a protocol that will not be merging to the main?  And is it a problem if a GHA tries to add a NEWS.md file in this case?  ;-)

Reply:
See [48608cf](https://github.com/inbo/protocolsource/pull/81/commits/48608cf0f0f814f0cb9837245c6127d09a97a08d), the extra `&& github.event.pull_request.base.ref == 'main'` should cover this.